packages/union-component/src/machines/donationType.ts

```diff
@@ -113,6 +113,7 @@ export type DonationMachineContext = {
     lastName: string;
     email: string;
     phoneNumber: string;
+    fund: string;
```

Comment:
@arathjz is this going to the request as `fund_id`?

Reply:
@orlando yes https://github.com/debtcollective/packages/blob/7c08aed49d1b4cf336912c769c8e354ae37d7a54/packages/union-component/src/api/donation.ts#L56